tests/test_uri_validate.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
# -*- coding: utf-8 -*- this line is not needed

Reply:
Thanks, I removed it.